src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -50,6 +51,27 @@ bool mpu6500WriteRegister(uint8_t reg, uint8_t data)
     return true;
 }
 
+bool verifympu6500WriteRegister(uint8_t reg, uint8_t data) {
+
+    uint8_t in;
+    uint8_t attemptsRemaining = 20;
+    static uint32_t loop = 0;
+    mpu6500WriteRegister(reg, data);
+    delayMicroseconds(100);
+
+    do {
+        mpu6500ReadRegister(reg, 1, &in);
+        if (in == data) {
+            return true;
+        } else {
+            mpu6500WriteRegister(reg, data);
+            debug[3] = loop++;
+            delayMicroseconds(1);
+        }
+    } while (attemptsRemaining--);
+    return false;
+}
+
```

Comment:
I see you fixed the two bugs I had in the MPU6000 version. A non returning bool and delay instead of delayMicroseconds. :)